tests/queries/0_stateless/03164_parts_filter_profiles_test.sql

```diff
@@ -0,0 +1,25 @@
+set log_queries=1;
+set log_queries_min_type='QUERY_FINISH';
+
+DROP TABLE IF EXISTS 03164_query_log_test_parts_filter_profiles;
+CREATE TABLE 03164_query_log_test_parts_filter_profiles
+(
+    a Int64, b Int64, c Int64,
+    INDEX idx_c(c) TYPE minmax
+) ENGINE = MergeTree PARTITION BY a ORDER BY b SETTINGS index_granularity=10;
+
+INSERT INTO 03164_query_log_test_parts_filter_profiles SELECT number, number+1000, number+10000 FROM numbers(100);
```

Comment:
Because of `PARTITION BY a` (l. 9) we'll obtain 100 partitions with a single row each, right? That is not realistic at all.

Let's remove the partition spec from l. 9, and instead create two parts (in a single implicit partition):

```sql
INSERT INTO 03164_query_log_test_parts_filter_profiles SELECT number, number+1000, number+10000 FROM numbers(100);
INSERT INTO 03164_query_log_test_parts_filter_profiles SELECT number, number+1000, number+10000 FROM numbers(100);
```

EDIT: Just a single test query is not comprehensive testing. Needs tests for:
- the basic case: no partitions, two parts
- a test with partitions: two partitions with two parts each
- a test with projections: are the metrics properly updated if a projection exists for a table?
- a test where no primary key is used